src/main/java/org/apache/mesos/hdfs/util/HDFSConstants.java

```diff
@@ -0,0 +1,23 @@
+package org.apache.mesos.hdfs.util;
+
+public class HDFSConstants {
+  // messages
+  public static final String NAME_NODE_INIT_MESSAGE = "-i";
+  public static final String NAME_NODE_BOOTSTRAP_MESSAGE = "-b";
+
+  // node ids
+  public static final String NAME_NODE_ID = "namenode";
+  public static final String JOURNAL_NODE_ID = "journalnode";
+  public static final String DATA_NODE_ID = "datanode";
+  public static final String ZKFC_NODE_ID = "zkfc";
+
+  // node task ids
+  public static final String JOURNAL_NODE_TASKID = ".journalnode.";
+  public static final String NAME_NODE_TASKID = ".namenode.namenode.";
```

Comment:
Because the namenode task is part of the namenode machine. It is marked as namenode.namenode. A journal node task that is part of the namenode machine would be marked as namenode.journalnode.